core/templates/pages/exploration-editor-page/translation-tab/translator-overview/translator-overview.component.spec.ts

```diff
@@ -58,6 +59,18 @@ describe('Translator Overview component', () => {
   let routerService: RouterService;
   let entityTranslationsService: EntityTranslationsService;
 
+  class MockUserExplorationPermissionsService {
+    getPermissionsAsync() {
+      return Promise.resolve({
+        canVoiceover: true
+      });
+    }
+
+    fetchPermissionsAsync() {
+      return Promise.resolve();
+    }
+  }
+
```

Comment:
Can we use spyOn? Or do we mock everywhere else?

Reply:
I have done some changes. PTAL.